gensim/models/word2vec.py

```diff
@@ -1869,3 +1873,15 @@ def __iter__(self):
         model.accuracy(args.accuracy)
 
     logger.info("finished running %s", program)
+    
+def methodize(func, instance):
```

Comment:
My [latest build](https://travis-ci.org/dust0x/gensim/builds/163828265) is failing on Python 2 for some reason, I'm working on that right now. Once that's done, I'll write the tests and merge it back with develop.